### PR TITLE
v2.2.0 release

### PR DESCRIPTION
**Pull request by ryanml**
This release adds support for the Sepolia network

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,10 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 ## [Unreleased]
+## [2.2.0] - 2022-09-19
+### Added
+- Add support for the Sepolia network ([#60](https://github.com/MetaMask/etherscan-link/pull/60))
+
 ## [2.1.0] - 2021-05-07
 ### Added
 
-- Add methods (getBlockExplorerLink, getAccountLink, getTokenTrackerLink) that select correct block explorer link format based on input params ([#31](https://github.com/MetaMask/etherscan-link/pull/44))
+- Add methods (getBlockExplorerLink, getAccountLink, getTokenTrackerLink) that select correct block explorer link format based on input params ([#44](https://github.com/MetaMask/etherscan-link/pull/44))
```
Comment (ryanml) [inline CHANGELOG.md:16]: Little link fix

**Review by NiranjanaBinoy — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (ryanml) [reply CHANGELOG.md]: @danjm no annoyance at all! :) Updated, will make a separate PR after this is merged.

**Review by danjm — APPROVED** [review]

**Comment by brad-decker** [comment]
> A new version of the package @metamask/etherscan-link (2.2.0) was published at 2022-09-19T18:24:33.162Z from
[[REDACTED]] The shasum of this package was 76314d0c1405a0669fc4a0a19e0877bd3d0c389f.

**Pull request merged**